httpx/models.py

```diff
@@ -794,6 +794,15 @@ def decoder(self) -> Decoder:
     def is_redirect(self) -> bool:
         return StatusCode.is_redirect(self.status_code) and "location" in self.headers
 
+    @property
```

Comment:
I don't think we should have these properties on the response object.

Reply:
Would you remove `is_redirect` as well? Maybe `is_client_error` is superfluous but seems like a valid case for redirects.

Reply:
Let's drop `expects_digest_auth` and `is_client_error`. I think they fall over the cutoff line.